### DISCUSSION TITLE
Move to pydantic 2.x

### DISCUSSION THREAD

**Discussion by theandrelima** [networktocode/network-importer#325] [General]
Hi team,
I wonder if there are any plans to move to pydantic 2.x. 

network-importer is an awesome tool, but the fact that is still depending on pydantic 1.6 is becoming a hindrance for anyone trying to use more modern tools that depend on pydantic 2.x above, as package managers like poetry will not be able to satisfy the requirements.

**Comment by jvanderaa** [comment]
There shouldn't be any impact when running in a separate environment. Many recommend going with a Virtual Environment to handle things for this.

Would support a merge of an update with pydantic 2.x. Really would love to remove it if we can as well.
